### DISCUSSION TITLE
How to show additional information in tooltip of time series panels

### DISCUSSION THREAD

**Discussion by sukhbir-singh** [grafana/grafana#71336] [Dashboards, Panels & Visualizations]
I need some help with tooltips on time series graph in Grafana. I am executing a kusto query which projects these 3 values: date_time (timestamp), data (int), and build_id (string).

Using this query, I am creating a time series graph. By default when i hover over any data point on the graph, i can only see data value as tool-tip. I also want to show build_id along with data on hover. In future, I might need to include more information in this tool tips information.

Any workaround to get this working in Grafana for now?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82604. Feel free to continue the discussion around this there. Thank you!
